camayoc/tests/qpc/ui/conftest.py

```diff
@@ -6,6 +6,13 @@
 from camayoc.utils import get_qpc_url
 
 
+def pytest_configure(config):
+    """Check for presence of driver for selenium."""
+    if not config.option.driver:
+        setattr(config.option, 'markexpr', 'not driver')
```

Comment:
I would use a different approach:

```python
def pytest_collection_modifyitems(config, items):
    if not config.getoption('--driver', None):
        skip_ui = pytest.mark.skip(
            reason='need --driver option to run UI tests')
        for item in items:
            if 'ui' in item.fspath.strpath:
                item.add_marker(skip_ui)
```

This way we don't need to inject any option and won't be overriding any option value passed by the user.

Bonus, by using the `getoption` method we can specify a default value to be returned if for some reason pytest-selenium is not installed.